packages/components/src/border-box-control/styles.ts

```diff
@@ -23,7 +23,8 @@ export const BorderBoxControlLinkedButton = (
 ) => {
 	return css`
 		flex: 0;
-		flex-basis: 36px;
+		flex-basis: 24px;
+		line-height: 22px;
```

Comment:
Not sure why 22px makes this work. It's there also in the legacy CSS.

Reply:
```suggestion
		line-height: 0;
```

> Not sure why 22px makes this work

I'm not sure this does work as `line-height: 22px` increases the button's wrapper height throwing off the overall control height and vertical alignment.

https://user-images.githubusercontent.com/60436221/188401934-ce9d9df3-4ff9-4523-9065-338835c31d1f.mp4